zeppelin-interpreter/src/main/thrift/RemoteInterpreterService.thrift

```diff
@@ -63,6 +65,16 @@ struct RemoteApplicationResult {
   2: string msg
 }
 
+struct ZeppelinServerResourceParagraphRunner {
+  1: string noteId,
+  2: string paragraphId
+}
+
+struct RemoteZeppelinServerController {
+  2: string eventOwnerKey
+  3: string msg
+}
```

Comment:
every message `RemoteInterpreter` send to ZeppelinServer using `RemoteInterpreterEventClient ` is RemoteInterpreterEvent type. Shell we avoid defining new type - ZeppelinServerResourceParagraphRunner and RemoteZeppelinServerController - for the payload ?
Because all other event are sending their payload using `RemoteInterpreterEvent` instead of defining their own struct in thrift idl. So let's do it in consistent way.